src/stactools/ephemeral/commands.py

```diff
@@ -0,0 +1,53 @@
+import click
+import logging
+
+from stactools.ephemeral import stac
+
+logger = logging.getLogger(__name__)
+
+
+def create_ephemeralcmd_command(cli):
+    """Creates the stactools-ephemeral command line utility."""
+    @cli.group(
+        "ephemeralcmd",
+        short_help=("Commands for working with stactools-ephemeral"),
+    )
+    def ephemeralcmd():
+        pass
+
+    @ephemeralcmd.command(
+        "create-collection",
+        short_help="Creates a STAC collection",
+    )
+    @click.argument("destination")
+    def create_collection_command(destination: str):
+        """Creates a STAC Collection
+
+        Args:
+            destination (str): An HREF for the Collection JSON
+        """
+        collection = stac.create_collection()
+
+        collection.set_self_href(destination)
+
+        collection.save_object()
+
+        return None
+
+    @ephemeralcmd.command("create-item", short_help="Create a STAC item")
+    @click.argument("destination")
+    @click.argument("source")
+    def create_item_command(destination: str, source: str):
```

Comment:
```suggestion
    @click.argument("source")
    @click.argument("destination")
    def create_item_command(source: str, destination: str):
```

I feel like the general convention is to put the source first and the destination second (see, e.g., `cp` and `gdal_translate`).